### PR TITLE
[#12048] Migrate FeedbackSessionClosedRemindersAction

### PR DESCRIPTION
**Pull request by EuniceSim142**
Part of #12048.

Closed emails are only sent to co-owner instructors.

### CODE REVIEW

File: src/main/java/teammates/sqllogic/core/FeedbackSessionsLogic.java
Comment (jayasting98) [inline src/main/java/teammates/sqllogic/core/FeedbackSessionsLogic.java:403]: Is it not possible to do this check through Hibernate?

**Review by jayasting98 — COMMENTED** [review]
Thanks for the work. I am just wondering about a few things related to Hibernate/SQL.

File: src/main/java/teammates/storage/sqlapi/FeedbackSessionsDb.java
Comment (EuniceSim142) [reply src/main/java/teammates/storage/sqlapi/FeedbackSessionsDb.java:260]: Good catch, that's my mistake

File: src/main/java/teammates/sqllogic/core/FeedbackSessionsLogic.java
Comment (EuniceSim142) [reply src/main/java/teammates/sqllogic/core/FeedbackSessionsLogic.java:403]: We could, but I think this restructures some of the existing logic and might change the semantics for what a method should be doing. Maybe leave this for a future optimization?

Comment (weiquu) [reply src/main/java/teammates/sqllogic/core/FeedbackSessionsLogic.java:403]: > We could, but I think this restructures some of the existing logic and might change the semantics for what a method should be doing. Maybe leave this for a future optimization?

+1 since we're quite pressed for time, the original way of implementation will do for now - let's focus more on correctness

**Review by weiquu — APPROVED** [review]
LGTM once merge conflicts are fixed!

**Pull request merged**
